vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -198,9 +237,18 @@ private void addCancelListenerForAddVariableDialog() {
 
   private void handleGroupSubmittedSuccess() {
     reloadExperimentalGroups();
+    if (hasExperimentalGroups) {
+      showSampleRegistrationPossibleNotification();
+    }
     experimentalGroupsDialog.close();
   }
 
+  private void showSampleRegistrationPossibleNotification() {
+    String projectId = this.context.projectId().map(ProjectId::value).orElseThrow();
+    Notification notification = createSampleRegistrationPossibleNotification(projectId);
```

Comment:
does not work as we need a different anchor every time.

Reply:
I'm confused to why though, the anchor should always stay the same upon loading the ExperimentMainComponent since it only needs the projectID which is already stored in the context.

Reply:
you might be right :) I prefer it this way. less possibility for side effects